06-reactjs/views/movie-creation-view.js

```diff
@@ -0,0 +1,120 @@
+// LIBS
+var React = require('react');
+var Router = require('react-router');
+var _ = require('lodash');
+
+// STORES
+var MovieStore = require('../store/movieStore');
+var MovieConstants = require('../store/movieConstants');
+
+// COMPONENTS
+var Input = require('../components/input');
+
+var MovieCreationView = React.createClass({
+
+    mixins: [ Router.Navigation ],
+
+    getInitialState: function () {
+        return this.getStateFromStore();
+    },
+
+    getStateFromStore(props) {
```

Comment:
This method should go on the getters section since is not part of the common react life cycle